src/constants/deck_ai_constants.asm

```diff
@@ -54,8 +55,24 @@ AI_ENERGY_TRANS_TO_BENCH EQU $e ; moves energy cards away from Arena card
 ; used to know which AI routine to call in
 ; the AIAction pointer tables in AIDoAction
 	const_def 1
-	const AIACTION_DO_TURN       ; $1
-	const AIACTION_START_DUEL    ; $2
-	const AIACTION_FORCED_SWITCH ; $3
-	const AIACTION_KO_SWITCH     ; $4
-	const AIACTION_TAKE_PRIZE    ; $5
+    const AIACTION_DO_TURN       ; $1
+    const AIACTION_START_DUEL    ; $2
+    const AIACTION_FORCED_SWITCH ; $3
+    const AIACTION_KO_SWITCH     ; $4
+    const AIACTION_TAKE_PRIZE    ; $5
```

Comment:
Seems like the indentation was changed to four spaces accidentally in these lines